README.md

```diff
@@ -54,6 +54,7 @@ Alternatively, `virtualenv` can be used by starting with:
 virtualenv -p python2.7 .virtualenv
 source .virtualenv/bin/activate
 ```
+Detailed installation instructions for Windows and Linux can be found [here.](Windows_Linux_Installation.txt)
```

Comment:
I would just clarify that this is for installing as a linux VM on windows vs. installing natively on windows